bookstore-scrapper/helpers/getGoogleBooksData.ts

```diff
@@ -0,0 +1,37 @@
+import { google, books_v1 } from "googleapis";
+import { GaxiosResponse } from "gaxios";
+
+export type GoogleBook = books_v1.Schema$Volume["volumeInfo"];
+
+type GoogleBooksVolume = books_v1.Schema$Volumes;
+
+type GoogleBooksItems = books_v1.Schema$Volumes["items"];
+
+const book = google.books({
+  version: "v1",
+});
+
+const timeout = (ms: number) => {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+};
+
+export const getGoogleBooksData = async (
+  isbn: string
+): Promise<GoogleBook | null> => {
+  /* Waiting on Google to provide us with a higher request/minute quota (currently capped at 100).
+     For now, it is doing a timeout for each request. */
+  await timeout(500);
```

Comment:
What you are doing here will wait half a second, then make the request. I just want to confirm that this is what you intended to do.

In terms of the delay time, 100 requests per minute works out to a request every 0.6s, so it might make sense to increase this delay. However, if it worked when you ran the indexing, then I am okay leaving the delay as is.

Reply:
Using API Key moving forwards. Resolved.